creator-node/sequelize/migrations/20200904183206-add-track-uuid-index.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addIndex('Files', ['trackUUID'], {
```

Comment:
needs to be `['type', 'trackUUID']`

Reply:
I think just trackUUID is the better move here. Adding type and trackUUID will only help *this* query and not others that need to query by trackUUID. Also since type is a small field, having the index on trackUUID will make the query much much faster

Reply:
i am wrong u r right